src/core-move/examples/sources/patterns/capability.move

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022, MoveFunsDAO.
+// SPDX-License-Identifier: Apache-2.0
+
+module examples::capability {
+    use std::signer;
+
+    const ENotPublisher: u64 = 0;
+    const ENotOwner: u64 = 1;
+
+    struct OwnerCapability has key, store {}
+
+    public entry fun init(sender: signer) {
+        assert!(signer::address_of(&sender) == @examples, ENotPublisher);
+        move_to(&sender, OwnerCapability {})
+    }
+
+    public fun grant_role_with_capability(_to: address, _cap: &OwnerCapability) {
+        // grant role to address
```

Comment:
`_to` should be `&signer` at here? how to keep the role?

Reply:
I just want to keep the example as simple as possible previously, now I've updated a new example with correct code.